build/common.props

```diff
@@ -3,7 +3,7 @@
     <!-- Packages can have independent versions and only increment when released -->
     <Version>3.0.31$(VersionSuffix)</Version>
     <ExtensionsStorageVersion>4.0.5$(VersionSuffix)</ExtensionsStorageVersion>
-    <HostStorageVersion>4.0.4$(VersionSuffix)</HostStorageVersion>
+    <HostStorageVersion>5.0.0-beta.1$(VersionSuffix)</HostStorageVersion>
```

Comment:
Since we're not building Extensions.Storage from this repo anymore, we should remove the previous ExtensionsStorageVersion line, and any other packaging code for extensions.